.github/workflows/release.yml

```diff
@@ -0,0 +1,60 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      deploy-to:
+        type: choice
+        description: Choose where to publish (test/prod)
+        options:
+        - prod
+        - test
```

Comment:
I used `PypiTest` and `PypiProd` for the environments with the trusted publishers.  Only because it matched what we use on the existing release for core + adapters.  We can update them to this but you'll need to do that in PyPI.  I'm not sure if it's editable.  You may need to delete and recreate the trusted publisher profile.